src/resources/styles.ts

```diff
@@ -331,7 +331,10 @@ export const haStyleDialog = css`
       );
       --mdc-dialog-min-height: 100%;
       --mdc-dialog-max-height: 100%;
+      /* both mdc-shape-medium and ha-card-border-radius affect the border radius
+        and should not take effect in fullscreen */
       --mdc-shape-medium: 0px;
+      --ha-card-border-radius: 0px;
```

Comment:
If this one is set, `--mdc-shape-medium` will not affect it anymore, so that is redundant

Reply:
Should not have an effect, but it felt safer to leave it in in case it affects something else in the mdc world.